src/lettuce/plugins/welcome.py

```diff
@@ -0,0 +1,29 @@
+from machine.plugins.base import MachineBasePlugin
+from machine.plugins.decorators import process
+
+
+class WelcomePlugin(MachineBasePlugin):
+    @process("member_joined_channel")
+    async def welcome(self, event):
+        user_id = event['user']
+        channel_id = event['channel']
+
+        welcome_message = self.get_welcome_message(user_id)
+
+        blocks = [
+            {
+                "type": "section",
+                "text": {
+                    "type": "mrkdwn",
+                    "text": welcome_message.strip()
+                }
+            }
+        ]
+
+        await self.say(channel=channel_id, text="Welcome to the OWASP Slack Community!", blocks=blocks)
+
+
+    def get_welcome_message(self, user_id):
```

Comment:
It's really expensive -- you should read the file template only once. I'd put it into the `__init__`